lib/classes/CLI.js

```diff
@@ -65,8 +65,14 @@ class CLI {
 
   generateMainHelp() {
     this.consoleLog('');
+    this.consoleLog(chalk.yellow('The Serverless Framework - https://serverless.com'));
+    this.consoleLog('');
+
+    this.consoleLog(chalk.yellow('Version:'));
+    this.consoleLog(version);
+    this.consoleLog('');
 
-    this.consoleLog(chalk.yellow.underline('Commands'));
+    this.consoleLog(chalk.yellow('Commands:'));
```

Comment:
I like the underline of it, as it clearly emphasises the different regions. Imho without underline the separation into different sections is less obvious.